model/trace/gen-ai.yaml

```diff
@@ -85,3 +83,33 @@ groups:
           conditionally_required: if and only if corresponding event is enabled
         note: >
           It's RECOMMENDED to format completions as JSON string matching [OpenAI messages format](https://platform.openai.com/docs/guides/text-generation)
+
+  - id: trace.gen_ai.client
```

Comment:
@lmolkova This is how I'm resolving the fact that *top_k* is not included in openai. Flagging it here for your review.